sentry_telegram/plugin.py

```diff
@@ -7,6 +7,7 @@
 from sentry.plugins.bases import notify
 from sentry.http import safe_urlopen
 from sentry.utils.safe import safe_execute
+from collections import defaultdict
```

Comment:
Thanks for contributing, @metheoryt!
Just one thing to apply your changes: let's make imports in the order "standard library - django - others - local imports".